modules/oe_theme_helper/tests/src/Kernel/RetinaScaleEffectTest.php

```diff
@@ -98,7 +98,7 @@ public function testRetinaScaleEffectForcedUpscaling(): void {
     $this->assertToolkitOperationsCalled(['scale']);
 
     $calls = $this->imageTestGetAllCalls();
-    $this->assertEquals($this->image->getWidth() * 2, $calls['scale'][0][0], 'Width is double the original size.');
+    $this->assertEquals($this->image->getWidth() * 2, $calls['scale'][0][0], 'Width must be twice the original size.');
```

Comment:
Same as above. Messages should reflect the failure, not the success. Removing them is OK too.